### ISSUE TITLE
rpmlint 2 complains about absence of %clean

### ISSUE THREAD

**Issue by jengelh** [rpm-software-management/rpmlint#537]
```
[   55s] libHX.spec: W: no-%clean-section
[   55s] The spec file doesn't contain a %clean section to remove the files installed
[   55s] by the %install section.
```

The `%clean` section is optional and implicit in rpm since a long time. rpmlint should not warn about it.

**Comment by Conan-Kudo** [comment]
Yeah, this warning should be turned off in Fedora and openSUSE policies. The reason it even still exists is because RPM upstream actually didn't start doing this until RPM 4.16.
